src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -65,6 +65,34 @@ class FileList extends React.Component {
     }
   }
 
+  ensureActiveItemVisible() {
+    if (!this.activeFileItem) {
+      return
+    }
+    
+    const currentClassName = !!this.fileList.current.className ? this.fileList.current.className : false;
+
+    if(!currentClassName){
+      return
+    }
+
+    // computeScrollIntoView thinks we need to scroll more elements than is actually necessary/desired
+    computeScrollIntoView(this.activeFileItem, { scrollMode: 'if-needed' }).forEach(({el, top }) => {
+      if (el.className !== currentClassName) {
```

Comment:
@digitalartcrew just do

`if (el.className !== this.fileList.current.className) {`